### PR TITLE
fix: browser address bar

### PR DESCRIPTION
**Pull request by frankvonhoven**
## **Description**
See more details [here](https://github.com/MetaMask/mobile-planning/issues/1519)

## **Related issues**

Fixes: [#1519](https://github.com/MetaMask/mobile-planning/issues/1519)

## **Manual testing steps**

1. See related issue
2.
3.

### **After**

https://github.com/MetaMask/metamask-mobile/assets/141057783/780bfed8-fd11-4aed-b88f-22de0a72bc4a

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/components/Views/BrowserTab/index.js [hunk 1]
```diff
@@ -1501,6 +1535,19 @@ export const BrowserTab = (props) => {
...
       >
         <View style={styles.webview}>
+          {progressBarPulseContainer()}
+          {!pageIsLoaded ? (
+            <View
+              style={{
+                justifyContent: 'center',
+                height: '100%',
+                backgroundColor: '#000000dd',
+                zIndex: 1,
```
Comment (tommasini) [inline app/components/Views/BrowserTab/index.js:1545]: I see, this view blocks the interaction on everything that's under it
Curious question, I see that there is  no opacity at this view, is this background color that gives that transparent look?

**Comment by tommasini** [comment]
Could you add recordings of android behaviour as well?

File: app/components/Views/BrowserTab/index.js (hunk 1 above)
Comment (frankvonhoven) [reply app/components/Views/BrowserTab/index.js:1545]: Yes! This is just a rough draft solution, but the last 2 digits of the hex code are the alpha for the background color

Comment (tommasini) [reply app/components/Views/BrowserTab/index.js:1545]: Thanks for the answer! Great work!

Comment (frankvonhoven) [reply app/components/Views/BrowserTab/index.js]: I updated it to use one of the theme colors 👍 that inline hex color was for POC

**Pull request closed without merging**